Updates to Charts (#180)
gristlabs/grist-help: help/widget-chart.md

## Chart types

> Each chart type plots several data series. Select the series to plot along the X-axis then choose to plot any other series by 

Reviewer: Some charts don't have an x-axis (pie and donut charts) so I'm suggesting a tweak to the copy. They also don't chart multiple series, so we can tweak that now.

```suggestion
Each chart type plots one or several data series. Select the series to plot by
```

> clicking the green 'Add Series' text in the [creator panel](glossary.md#creator-panel). If adding more than one series, the 

Reviewer: ```suggestion
clicking the green 'Add Series' text in the [creator panel](glossary.md#creator-panel). 
```
^ This is a hold over from the old way labels were selected. Now, we have a dedicated space in the UI for selecting X-axis and labels.

> order of the list matters and determines how each series is used, as described below.

Reviewer: ``` suggestion
In the area above the "Series" section of the creator panel, you may configure the x-axis for most charts, or the labels for pie and donut charts.
```

#### Bar Chart

> Needs two or more series: the first for bar labels (x-axis), the second for bar heights (y-axis). Additional series create 

Reviewer: ```suggestion
Needs an x-axis and at least one series to plot along the y-axis. Additional series create 
```

This change needs to be carried out to all the charts that talk about a need for two series minimum, such as the line chart. labels for pie/donut charts, so on. It's no longer the case that x-axis is set as the first series; it has a dedicated dropdown.

Author: Did not update Scatter Plot or Kaplan-Meier Plot because the x-axis dropdown is not actually for x-axis, it's just for labels, a bit misleading.

> additional bars at each data point and specify their heights.

Reviewer: ```suggestion
additional bars at each point on the x-axis. To stack series onto the same bar, select the "Stack series" checkbox.

## Chart types

Each chart type plots one or several data series. Select the series to plot by 
clicking the green 'Add Series' text in the [creator panel](glossary.md#creator-panel). In the area above the "Series" section of the creator panel, you may configure the x-axis for most charts, or the labels for pie and donut charts.

#### Bar Chart

Needs an x-axis and at least one series to plot along the y-axis. Additional series create 
additional bars at each point on the x-axis. To stack series onto the same bar, select the "Stack series" checkbox.

*![widget_chart_bar](images/widget_chart_bar.png)*

#### Line Chart

Needs an x-axis and at least one series to plot along the y-axis. 
Additional series specify Y values for additional lines.

*![widget_chart_line](images/widget_chart_line.png)*

#### Pie Chart

Needs pie slice labels and one series for the pie slice sizes.

*![widget_chart_pie](images/widget_chart_pie.png)*

#### Area Chart

Similar to a line chart, needs an x-axis and at least one series to plot along the y-axis. 
Additional series specify Y values for additional lines.

*![widget_chart_area](images/widget_chart_area.png)*

#### Scatter Plot

> Needs three or more series: the first for the point labels, and the second and third for the X and Y values for each point, respectively. Additional series specify Y values for additional sets of points.

Reviewer: ```suggestion
Needs three or more series: the first for the point labels, and the second and third for the X and Y values for each point, respectively. Additional series specify Y values for additional sets of points.
```

Needs a label and two or more series. The label applies to the points. The series apply the X and Y values for each point, respectively. Additional series specify Y values for additional sets of points.

belongs, the second giving a survival time or time-to-failure of that point. The plot shows the

> survival times on the X axis, and the number of points that survive at that time on the Y axis.

Reviewer: Need to update the text to reflect the change in the UI. Suggestion:

The [Kaplan-Meier Plot](https://en.wikipedia.org/wiki/Kaplan%E2%80%93Meier_estimator) is useful for certain studies, and needs one label and one series. The label applies to the lines being plotted. The series gives a survival time or time-to-failure of that point. The plot shows the
survival times on the X axis, and the number of points that survive at that time on the Y axis.

> A number of chart options are available, some of them specific to certain chart types.

Reviewer: Might want to add "Stack series" to this list. It's for bar charts.

**Show markers** [for Line Charts only]: When checked, each point on the line is marked

> additionally by a small circle. See the example for "Group by first column" above.

Reviewer: ```suggestion
additionally by a small circle. See the example for Split Series above.
```
